src/v2/Components/Artwork/FillwidthItem.tsx

```diff
@@ -118,6 +124,10 @@ export class FillwidthItemContainer extends React.Component<
       return null
     }
 
+    const name = artwork?.artistNames ?? ""
+    const title = artwork?.title ?? ""
+    const alt = name === "" ? "" + title : name + ", " + title
```

Comment:
Also here I think we use `imageTitle` as well if having the year in the `alt` tag isn't some SEO anti-pattern. Not sure why it would be, but its safer to double check with an SEO person.

Reply:
I think `imageTitle` is fine to use here!